src/main/java/cart/service/CartService.java

```diff
@@ -67,14 +59,8 @@ public void deleteCartItem(String email, Long cartItemId) {
                 .orElseThrow(() -> new ItemNotFoundException("장바구니에서 일치하는 상품을 찾을 수 없습니다."));
 
         Cart cart = cartRepository.findCart(user);
-        validateNotExistsItem(cart, item);
-
-        cartRepository.deleteCartItem(cart, item);
-    }
+        cart.removeItem(item);
 
-    private void validateNotExistsItem(Cart cart, Item item) {
-        if (!cart.isExistsItem(item)) {
-            throw new CartNotFoundException("장바구니에 존재하지 않는 상품입니다.");
-        }
+        cartRepository.deleteCartItem(cart);
```

Comment:
```suggestion
        cartRepository.save(cart);
```

save 로직으로 처리할 수 있지 않을까요?

Reply:
이전에 보여주신 `baeldung 카산드라 DB` 예제를 보고 저도 `save` 하나로 처리할까 하다가, 명확한게 좋지 않나 싶어서 `save`와 `delete`를 분리했었습니다.

어처피 `insert`나 `delete`나 모두 데이터를 변경하기도 하고, `save`의 의미를 `현재 상태의 장바구니를 저장한다`라고 해석할 수도 있겠네요.

말씀해주신대로 리펙토링을 진행해보도록 하겠습니다! 감사합니다!